src/aggregator/aggregator/forwarded_writer.go

```diff
@@ -474,8 +474,8 @@ func (agg *forwardedAggregation) onDone(key aggregationKey) error {
 }
 
 func (agg *forwardedAggregation) index(key aggregationKey) int {
-	for i, k := range agg.byKey {
-		if k.key.Equal(key) {
+	for i := range agg.byKey {
+		if agg.byKey[i].key.Equal(key) {
```

Comment:
Dumb question: this is faster? I'm surprised haha--more inlining issues?

Reply:
stack copies due to how range works

Reply:
Ah nice, TIL.